_docs/quickstarts/scanner/descriptions.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Scanner Descriptions and Dependencies
+order: 102
+---
+
+# {{ page.title }}
+
+This page describes the Forseti scanners that are available, how they work, and
+why they're important. You can configure Scanner to execute multiple scanners in
+the same run. Learn about [configuring
+Scanner]({% link _docs/howto/configure/configuring-forseti#configuring-scanner %}).
```

Comment:
To do this properly (and not break liquid templates you'll need to add the heading level link outside the {}, e.g.:

`({% link _docs/howto/configure/configuring-forseti.md %}#configuring-scanner)`